### PR TITLE
Added embed site via chat plugin

### PR DESCRIPTION
**Pull request by divyam-rai**

### CODE REVIEW

File: embed-site-via-chat/src/App.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import { Chat, Streams, Users } from '@livestorm/plugin'
+const template = require('./iframe.html').default
+
+export default async function() {
+  const connected_user = await Users.me();
```
Comment (Michaelvilleneuve) [inline embed-site-via-chat/src/App.ts:5]: You'll find some properties that are not camel (when they come from the backend) but in general we try to use camel everywhere `connectedUser`

File: embed-site-via-chat/src/App.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+  function sendChatError(text){
+    Chat.send({
+      user: {
+        firstName: 'Livestorm',
+        lastName: 'Bot',
+        tag: 'Sent privately to you'
+      },
+      text: `${text}`
```
Comment (Michaelvilleneuve) [inline embed-site-via-chat/src/App.ts:14]: Is there a reason why you did not just:
```javascript
Chat.send({
      user: {
        firstName: 'Livestorm',
        lastName: 'Bot',
        tag: 'Sent privately to you'
      },
      text
```

No need to interpolate it I think

File: embed-site-via-chat/src/App.ts [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    });
+  }
+
+  function createEmbedStream(url){
+    var template_options = {
```
Comment (Michaelvilleneuve) [inline embed-site-via-chat/src/App.ts:19]: Same here regarding the case : 
```
const templateOptions = 
```

File: embed-site-via-chat/src/App.ts [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+      title : 'Embed Stream',
+      imageUrl: '',
+      template: template,
+      variables: { url: url },
```
Comment (Michaelvilleneuve) [inline embed-site-via-chat/src/App.ts:23]:
```javascript
variables: { url },
```

File: embed-site-via-chat/src/App.ts [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+      },
+      text: `${text}`
+    });
+  }
+
+  function createEmbedStream(url){
+    var template_options = {
+      title : 'Embed Stream',
+      imageUrl: '',
+      template: template,
```
Comment (Michaelvilleneuve) [inline embed-site-via-chat/src/App.ts:22]:
```javascript
template,
```

File: embed-site-via-chat/src/App.ts [hunk 6]
```diff
@@ -0,0 +1,52 @@
+import { Chat, Streams, Users } from '@livestorm/plugin'
+const template = require('./iframe.html').default
+
+export default async function() {
+  const connected_user = await Users.me();
+
+  function sendChatError(text){
```
Comment (Michaelvilleneuve) [inline embed-site-via-chat/src/App.ts:7]: Very minor but in general we tend to leave a space after the function :
```javascript
function sendChatError(text) {
```
you have some other examples below

File: embed-site-via-chat/src/App.ts (hunk 1 above)
Comment (divyam-rai) [reply embed-site-via-chat/src/App.ts]: Just added a commit fixing this.

File: embed-site-via-chat/src/App.ts (hunk 2 above)
Comment (divyam-rai) [reply embed-site-via-chat/src/App.ts]: Just added a commit fixing this.

File: embed-site-via-chat/src/App.ts (hunk 3 above)
Comment (divyam-rai) [reply embed-site-via-chat/src/App.ts]: Just added a commit fixing this.

File: embed-site-via-chat/src/App.ts (hunk 4 above)
Comment (divyam-rai) [reply embed-site-via-chat/src/App.ts]: Just added a commit fixing this.

File: embed-site-via-chat/src/App.ts (hunk 5 above)
Comment (divyam-rai) [reply embed-site-via-chat/src/App.ts]: Just added a commit fixing this.

File: embed-site-via-chat/src/App.ts (hunk 6 above)
Comment (divyam-rai) [reply embed-site-via-chat/src/App.ts]: Just added a commit fixing this.

**Review by Michaelvilleneuve — APPROVED** [review]

**Pull request merged**